docs/Configuration.md

```diff
@@ -94,17 +95,16 @@ Some general information:
 
 #### builder (optional)
 - `resources`: General resource configuration for this project
-    - `excludes`: List of glob patterns. Matching resources of this project will be ignored by the builder (i.e. all tasks).  
```

Comment:
I think those two spaces were put there intentionally to have a line-break

Reply:
Done. I have put them back to keep the line-break